lib/sycamore/sycamore/connectors/duckdb/duckdb_reader.py

```diff
@@ -0,0 +1,76 @@
+from sycamore.data import Document
+
+from dataclasses import dataclass
+from typing import Optional
+
+from sycamore.connectors.base_reader import BaseDBReader
+import duckdb
+
+
+@dataclass
+class DuckDBReaderClientParams(BaseDBReader.ClientParams):
+    pass
+
+
+@dataclass
+class DuckDBReaderQueryParams(BaseDBReader.QueryParams):
+    db_url: str
```

Comment:
slight preference for putting db_url in ClientParams but either way

Reply:
I agree generally but in DuckDB since it initializes the connection quickly I'd prefer starting the work as late as possible